### PR TITLE
Package the HIP/ROCm and ROCrand libraries [13.0.x]

### PR DESCRIPTION
**Pull request by fwyzard**
Package the core HIP/ROCm libraries and tools, and the ROCrand libraries.
Update the tool files for rocm-rocrand and alpaca-rom accoridngly.

### CODE REVIEW

**Comment by fwyzard** [comment]
backport #8370

File: cmssw-tool-conf.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### RPM cms cmssw-tool-conf 53.0
+### RPM cms cmssw-tool-conf 53.1
```
Comment (fwyzard) [inline cmssw-tool-conf.spec:1]: @smuzaffar is this the right way to have different `cmssw-tool-conf` files for CMSSW 13.0.x vs 13.1.x ?

**Comment by fwyzard** [comment]
please test

File: cmssw-tool-conf.spec (hunk 1 above)
Comment (smuzaffar) [reply cmssw-tool-conf.spec:1]: Normally I just increment the major version when a new tool is added

Comment (fwyzard) [reply cmssw-tool-conf.spec:1]: Yes... but `54.0` is already used in 13.1.x, which adds tools I think we do not want to backport to 13.0.x.

So I've used `55.0` for the 13.1.x branch, and `53.1` for the 13.0.x branch.

Comment (smuzaffar) [reply cmssw-tool-conf.spec:1]: ah ok,  sorry I did not realize that it is 13.0.x PR. yes `53.1` seems right

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
please test for el9_amd64_gcc11

**Comment by fwyzard** [comment]
please test for slc7_amd64_gcc11

**Comment by rappoccio** [comment]
@smuzaffar should we merge this one now? There are IB failures in the 13_1 but these seem unrelated to this PR.

**Comment by smuzaffar** [comment]
+externals

looks good to go in 13.0.X

**Comment by smuzaffar** [comment]
yes @rappoccio , this is ready to go in 13.0.X

**Comment by perrotta** [comment]
merge

**Pull request merged**